transformations/negated_antonym_perturbation/README.md

```diff
@@ -0,0 +1,27 @@
+# Negated Antonym Perturbation 
+This perturbation rephrases the adjectives and adverbs to their negated antonym
+
+Authors: Mayukh Das (Technical University of Braunschweig  / mayukh.das@tu-bs.de) 
+
+## What type of a transformation is this?
+This transformation types of adjectives and adverbs and converts them to their negated antonym.
+Therefore, the transformation retains the semantics of the original text as positives gets converted to negated negatives and negatives gets converted to negated positives.
+Example: I think you are prepared for the test --> I think you are not unprepared for the test
+## What tasks does it intend to benefit?
+This can act as a valid perturbation that retains the semantics.
+This will benefit for robustness of text classification like sentiment analysis, etc.
+
+
+## What are the limitations of this transformation?
```

Comment:
@kaustubhdhole unfortunately yes except for the not word as  @marco-digio pointed. As the addition of 'not' gives a better robustness score than #222 (I have only checked for roberta-base-imdb) , It could be merged as a special case of the other PR if suited.